frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx

```diff
@@ -78,15 +78,14 @@ export function OperatorValueSelect({
 
     const [operators, setOperators] = useState([] as Array<PropertyOperator>)
     useEffect(() => {
-        const isAutocaptureElementProperty = propkey === 'selector'
         const operatorMapping: Record<string, string> = chooseOperatorMap(
-            isAutocaptureElementProperty ? PropertyType.Selector : propertyDefinition?.property_type
+            propkey === 'selector' || propkey === 'tag_name' ? PropertyType.Selector : propertyDefinition?.property_type
```

Comment:
This was wrong. Not only `selector`, but also `tag_name` only support a narrowed down list of options (just `==` and `!=`). Instead of adding `PropertyType.TagName`, I did this hack, as `PropertyType` is something that's synced from the plugin server's types 🤯. We'll need to revisit this anyway, so I hope the fix is good enough...